lib/rubocop/ast/builder.rb

```diff
@@ -14,6 +14,8 @@ module AST
     #   parser = Parser::Ruby25.new(builder)
     #   root_node = parser.parse(buffer)
     class Builder < Parser::Builders::Default
+      self.emit_forward_arg = true
```

Comment:
Shouldn't we leave it to RuboCop to set this, for the sake of consistency in `rubocop-ast`?

Reply:
Contrary to the other `emit_`, this is really fixing an error in the AST. We are < 1.0 so I think it's best to set this default now.

Reply:
Fair enough.